### DISCUSSION TITLE
How do we characterize UDP vs TCP streaming performance?

### DISCUSSION THREAD

**Discussion by castroflavio** [scistream/scistream-proto#38] [General]
What are the streaming overheads for UDP vs TCP streaming?

**Comment by JoaquinChung** [comment]
I would say like everything... it depends. If the scientific application tolerates packet losses, maybe we can use UDP transport and get away with low overhead. Otherwise, a reliable transport (e.g., QUIC) must be implemented on top of UDP, and then we can compare overhead apples to apples with TCP
